draft-packages/button/KaizenDraft/styles.scss

```diff
@@ -0,0 +1,451 @@
+@import "~@kaizen/deprecated-component-library-helpers/styles/color";
```

Comment:
I know we briefly spoke about this but is severing this color dependency too ambitious for this PR?

Reply:
I would like to, but I'm not comfortable with the risk of silently introducing visual bugs. Would prefer to do this after we have dart-sass in Murmur.